### PR TITLE
refactor: Redesigned projects built with MACI page

### LINKED ISSUE #1613: re-design projects built with MACI page
Redesign https://maci.pse.dev/projects to look nicer and be easily amended when new projects are available: can ideally re-use semaphore's website code: 
https://semaphore.pse.dev/projects -> https://github.com/semaphore-protocol/semaphore/tree/main/apps/website

### PR DESCRIPTION
**Pull request by MukulKolpe**
# Description

Redesigned the MACI projects page to improve user interface and make it easier to update when new projects are added.

https://github.com/user-attachments/assets/3eb40c77-260c-4e21-9b12-08f4f4dafb1c

## Related issue(s)

Fixes #1613

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

**Comment by MukulKolpe** [comment]
Hey @ctrlc03, sorry for the delay. Please let me know if any adjustments are needed regarding themes, wording, conventions, etc.

File: apps/website/src/content/projects.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    "name": "MACI_QF in Allo",
+    "description": "a MACI Quadratic Funding strategy to be used within Gitcoin's Allo Protocol.",
+    "hackathon": null,
+    "status": "Production",
+    "links": {
+      "github": "https://github.com/gitcoinco/MACI_QF"
+    }
+  },
+  {
+    "name": "MACI-RPGF",
```
Comment (ctrlc03) [inline apps/website/src/content/projects.json:23]: can we change this to MACI Platform

File: apps/website/src/content/projects.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    "description": "a fork from Easy-RPGF that enables any community to easily run an RPGF round using MACI.",
```
Comment (ctrlc03) [inline apps/website/src/content/projects.json:24]: can we change this to: a platform that allows to run different types of vote and funding rounds using MACI.

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
This looks amazing thank you very much 🚀 

Could you please ensure linter checks are passed (you can run locally with `pnpm lint:ts:fix`) and also it seems like navigation buttons are not displaying correctly in light mode, and the cards and text also

**Comment by kittybest** [comment]
I think the light mode color should be adjusted, other things are perfect! thanks! 

image

**Comment by MukulKolpe** [comment]
Thank you @ctrlc03 @kittybest! Please let me know if any other changes are needed.

https://github.com/user-attachments/assets/bb9baba3-9768-4201-a92f-2c4368bdb3c4

**Comment by ctrlc03** [comment]
@MukulKolpe seems like you used merge rather than rebase, please squash your commits and rebase to the latest dev :)

**Review by kittybest — APPROVED** [review]
It looks so nice, thanks a lot!

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -12112,6 +12112,35 @@ snapshots:
       - vue-template-compiler
       - webpack-cli
 
+<<<<<<< HEAD
+=======
```
Comment (ctrlc03) [inline pnpm-lock.yaml:12116]: there's conflict markers here

**Pull request merged**